### ISSUE TITLE
MULTI GPU question

### ISSUE THREAD

**Issue by pepi99** [yangheng95/PyABSA#164]
My code is:

Trainer(config=apc_config_english,
        dataset=CryptoDataset,  # train set and test set will be automatically detected
        checkpoint_save_mode=2,  # =None to avoid save model
        auto_device=True  # automatic choose CUDA or CPU
        )

If I run it on a server with 4 GPUs, is it going to perform computation on ALL GPUS, or ONLY one? 

If Only one, how can I perform on all 4?
Regards

**Comment by yangheng95** [comment]
Please try set auto_device='allcuda'

**Comment by pepi99** [comment]
OK THANKS! I will try tomorrow and tell you.

**Comment by pepi99** [comment]
works.
